index-providers/modeshape-elasticsearch-index-provider/src/main/java/org/modeshape/jcr/index/elasticsearch/EmptyResults.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * ModeShape (http://www.modeshape.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.modeshape.jcr.index.elasticsearch;
+
+import org.modeshape.jcr.spi.index.Index;
+import org.modeshape.jcr.spi.index.ResultWriter;
+
+/**
+ *
+ * @author kulikov
+ */
+public class EmptyResults implements Index.Results {
```

Comment:
This class it not needed. Instead, [this](https://github.com/okulikov/modeshape/blob/MODE-2162-b/modeshape-jcr/src/main/java/org/modeshape/jcr/index/local/Operations.java#L57) should be "refactored out" and placed as a constant (`public static final`) in the `Results` interface from the SPI.